src/sentry/conf/server.py

```diff
@@ -3468,8 +3469,52 @@ def build_cdc_postgres_init_db_volume(settings: Any) -> dict[str, dict[str, str]
 
 # spans
 KAFKA_SNUBA_SPANS = "snuba-spans"
+# END DEPRECATED SECTION
+
+
+# Mapping of default Kafka topic name to custom names
+KAFKA_TOPIC_OVERRIDES: Mapping[str, str] = {
+    "generic-metrics-subscription-results": KAFKA_GENERIC_METRICS_SUBSCRIPTIONS_RESULTS
```

Comment:
Temporary so the override works for this topic which is the only one defined in the new-style. Later this will be removed when all overrides are defined in production

Reply:
I am not following why we need to set up the override this way for one topic only. Could you please provide some more context ?

Reply:
Two reasons
- demonstration of how this works
- it is the only topic that has schema validation turned on. In order to keep that happening, we need to know both the default and override topic there so it must use the new system.

Reply:
ok